src/PyHyperScattering/SST1RSoXSLoader.py

```diff
@@ -112,7 +118,7 @@ def loadSingleImage(self,filepath,coords=None, return_q=False,image_slice=None,u
 
         # # step 2: dark subtraction
         # this is already done in the suitcase, but we offer the option to add/subtract a pedestal.
-        image_data = (np.array(img)-self.dark_pedestal)/corr
+        image_data = (img-self.dark_pedestal)/corr
         if return_q:
             qpx = 2*np.pi*60e-6/(headerdict['sdd']/1000)/(headerdict['wavelength']*1e10)
             qx = (np.arange(1,img.size[0]+1)-headerdict['beamcenter_y'])*qpx
```

Comment:
CI is returning an error in making this qx array, since img.size is now an int since img is now a `np.ndarray`.  This should be an easy fix, I think:
```suggestion
            qx = (np.arange(1,img.shape[0]+1)-headerdict['beamcenter_y'])*qpx
            qy = (np.arange(1,img.shape[1]+1)-headerdict['beamcenter_x'])*qpx
```